hub_map.html

```diff
@@ -491,15 +496,25 @@ <h2>Sunrise Hubs</h2>
       });
   }
 
-  function groupLinkHtml(groupLink) {
-    // TODO: rename spreadsheet column in original CSV to account for non-facebook links
-    if (/facebook\.com/i.test(groupLink)) { // Facebook
-      return '<p><a href="'+ groupLink + '" target="_blank" style="text-decoration:underline;">Facebook Page</a></p>';
-    } else if (groupLink.trim().startsWith("@")) { // Twitter
-      return '<p><a href="https://twitter.com/'+ groupLink.trim().slice(1) + '" target="_blank" style="text-decoration:underline;">Twitter Page</a></p>';
-    } else { // Other website
-      return '<p><a href="'+ groupLink + '" target="_blank" style="text-decoration:underline;">Hub Website</a></p>';
+  function groupLinkHtml(properties) {
+    var html = '';
+    var fbLink = properties.facebookGroupLink;
+    var twLink = properties.twitterLink;
+    var wsLink = properties.websiteLink;
+    if (fbLink) {
+      html += '<p><a href="'+ fbLink + '" target="_blank" style="text-decoration:underline;">Facebook Page</a></p>';
+    }
+    if (twLink) {
+      if (twLink.startsWith("@")) {
+        html += '<p><a href="https://twitter.com/'+ twLink.slice(1) + '" target="_blank" style="text-decoration:underline;">Twitter Page</a></p>';
+      } else {
+        html += '<p><a href="'+ twLink + '" target="_blank" style="text-decoration:underline;">Twitter Page</a></p>';
+      }
     }
+    if (wsLink) {
+      html += '<p><a href="'+ wsLink + '" target="_blank" style="text-decoration:underline;">Hub Website</a></p>';
+    }
```

Comment:
@skyfaller, I know you mentioned that if the hub has a website, it might make sense to only link to the website rather than other social media. However, I noticed that (for the few hubs which have websites), their social media accounts were often much more active, and also not linked to from their websites. So I was thinking we could just ask the hubs what they prefer when we email them. We could also have special treatment for `sunrisemovement.org` hub websites.